Packs/DSPM/Integrations/DSPM/DSPM.yml

```diff
@@ -0,0 +1,495 @@
+      type: String
+  - arguments:
+    - description: ID of the asset for which to retrieve file details.
+      name: asset_id
+      required: true
+    description: Retrieves file details for the specified asset ID.
+    name: dspm-get-asset-files-by-id
+    outputs:
+    - contextPath: DSPM.AssetFiles
+      description: File Details of the asset for the provided ID.
+      type: String
+  - arguments:
+    name: dspm-get-data-types
+    outputs:
+    - contextPath: DSPM.DataTypes
+      description: List of data types for company.
+      type: String
+  - arguments:
+    - description: List of regions (comma-separated values)
+      isArray: false
+      isOptional: true
+      name: regionIn
+      required: false
+      secret: false
+    - description: Exact region
+      isArray: false
+      isOptional: true
+      name: regionEqual
+      required: false
+      secret: false
+    - description: 'List of cloud providers ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"]'
+      isArray: false
+      isOptional: true
+      name: cloudProviderIn
+      required: false
+      secret: false
+    - description: 'Exact cloud provider ["AWS", "AZURE", "GCP", "SNOWFLAKE",
+      "FILE_SHARE", "O365"]'
+      isArray: false
+      isOptional: true
+      name: cloudProviderEqual
+      required: false
+      secret: false
+    - description: List of Service Types (comma separated values)
+      isArray: false
+      isOptional: true
+      name: serviceTypeIn
+      required: false
+      secret: false
+    - description: Exact Service Type
+      isArray: false
+      isOptional: true
+      name: serviceTypeEqual
+      required: false
+      secret: false
+    - description: List of Life Cycles (comma separated values)
+      isArray: false
+      isOptional: true
+      name: lifecycleIn
+      required: false
+      secret: false
+    - description: 'List of project IDs'
+      isArray: true
+      isOptional: true
+      name: projectIdIn
+      required: false
+      secret: false
+    - description: 'Exact project ID'
+      isArray: false
+      isOptional: true
+      name: projectIdEqual
+      required: false
+      secret: false
+    - description: Exact Life Cycle
+      isArray: false
+      isOptional: true
+      name: lifecycleEqual
+      required: false
+      secret: false
+    - description: 'Sorting criteria in the format: property,(asc|desc). Default sort
+      order is ascending. Multiple sort criteria are supported.'
+      isArray: false
+      isOptional: true
+      name: sort
+      required: false
+      secret: false
+    description: Retrieves a list of data type findings for the company.
+    name: dspm-get-data-types-findings
+    outputs:
+    - contextPath: DSPM.DataTypeFindings
+      description: List of data type findings for the company.
+      type: String
+  # - arguments: []
+  #   description: for testing the fetch-handle method.
+  #   name: fetch-incidents
```

Comment:
```suggestion
```

Reply:
We will remove this commented code from yml.